megamek/src/megamek/common/Board.java

```diff
@@ -1020,6 +1020,10 @@ public void load(InputStream is, @Nullable List<String> errors, boolean continue
                         args[i++] = st.ttype == StreamTokenizer.TT_NUMBER ? (int) st.nval + "" : st.sval;
                     }
                     int elevation = Integer.parseInt(args[1]);
+                    if (mapType == T_ATMOSPHERE) {
+                        // All foliage on low-altitude maps have height 1
+                        args[2] = args[2].replaceAll("foliage_elev:\\d{1,}", "foliage_elev:1");
```

Comment:
Won't this affect graphics? Showing all woods as "foliage" (low bushes)
It is sad that we actually have Terrains.getTerrainElevation(...) that already retrieves the height depending on if the map is atmospheric. It doesnt seem to be used in LosEffects.
Also, I think this change would much more naturally fit into Board.initializeFoliageElev(). This does post load processing anyway.

Reply:
It does reduce the graphics to foliage, although I personally found it a good reminder that the "level" would only be one higher.

I looked at getTerrainElevation() but wasn't sure that it would meet our needs.  I'll re-examine it, and look at initializeFoliageElev().

Reply:
Removed this stuff; unnecessary.